Cloud Pak for Data/WML/notebooks/model risk management/OpenScale MRM Cloud pak for Data.ipynb

```diff
@@ -15,7 +15,7 @@
     "id": "4f7234e8-ee83-41dd-9596-89b0b50274fe"
    },
    "source": [
-    "This notebook should be run using Default Spark 3.0 & Python 3.8 runtime environment. If you are viewing this in Watson Studio and do not see Python 3.8.x with Spark in the upper right corner of your screen, please update the runtime now. It requires service credentials for the following services:\n",
+    "This notebook should be run using Default Spark 3.0 & Python 3.9 runtime environment. If you are viewing this in Watson Studio and do not see Python 3.9.x with Spark 3.0.x in the upper right corner of your screen, please update the runtime now. It requires service credentials for the following services:\n",
```

Comment:
The Spark version should be changed to 3.2 because that's what is given below `spark-mllib_3.2`